deploy/role.yaml

```diff
@@ -90,3 +90,104 @@ rules:
   - patch
   - update
   - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: cincinnati-operator
+rules:
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - images
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
I don't know if we care to limit it but I think you can also limit it to a particular resourceName.

Reply:
I'm fine either way.  This is a singleton resource, so we're pretty much limited out of the box.